src/GameSlice.js

```diff
@@ -74,6 +71,14 @@ export const GameSlice = createSlice({
       state.currentSeconds += 1;
     },
     getDailyKryptoSuccess: (state, action) => {
+      if (state.kryptoId !== action.payload.id) {
+        state.currentSeconds = 0;
+        state.playedToday = false;
```

Comment:
This will show the how to play menu every day for the user. Is that what you want?

Reply:
For now, yah